src/physics/base/PhysicsStepUtils.i.hh

```diff
@@ -135,14 +138,33 @@ CELER_FUNCTION ModelId select_model(const ParticleTrackView& particle,
     }
```

Comment:
In an earlier discussion, I think you suggested and I agreed it would make more sense to only call this function if the particle is interacting:
```suggestion
    CELER_EXPECT(physics.interaction_mfp() > 0);
```

Reply:
humm did you mean
```
CELER_EXPECT(physics.interaction_mfp() <= 0);
```